src/comments.ts

```diff
@@ -91,3 +91,8 @@ I can't [accept a merge request](${uri}) until the PR has a green CI and was app
 
 Thanks, and happy typing!`
 });
+
+export const Suggestions = (user: string) => ({
+    tag: "suggestions",
+    status: `@${user} I noticed these differences from the required form. If you can revise your changes to avoid them, so much the better! Otherwise please reply with explanations why they're needed (unless it's obvious) and a maintainer will take a look. Thanks!`,
```

Comment:
Because this applies to `tsconfig.json` and `package.json`, as well as `tslint.json`, I'd like to see the current language in action and how people react to it first, then revisit based on the observed shortcomings.

Reply:
OK, so maybe just drop the `(unless it's obvious)` comment?  If it's really obvious, people will most likely not bother with it anyway.

Reply:
Done :heavy_check_mark: